### DISCUSSION TITLE
How can i make mineflayer bot auto eat golden apple

### DISCUSSION THREAD

**Discussion by Foodnunglz** [PrismarineJS/mineflayer#3115] [Q&A]
I want to activate auto eat golden apple when bot's health bar is less than 18

**Comment by PondWader** [comment]
Use `bot.consume` to eat the apple (https://github.com/PrismarineJS/mineflayer/blob/master/docs/api.md#botconsume) when it's holding it. 

You can use the "health" event to detect when the bots health changes and use bot.health to check the health.
